improver/cli/__init__.py

```diff
@@ -212,7 +212,7 @@ def create_constrained_inputcubelist_converter(*constraints):
     These cubes get loaded and returned as a CubeList.
 
     Args:
-        *constraints (tuple of str or list):
+        *constraints (callable):
```

Comment:
In the docstring above, I think it should read "So an example is if you wanted..."